roles/ngi_reports/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 ngi_reports_repo: https://github.com/NationalGenomicsInfrastructure/ngi_reports.git
 ngi_reports_dest: "{{ sw_path }}/ngi_reports"
-ngi_reports_version: fab25b4d3f81744b02584b301ce610e58e853c50
+ngi_reports_version: 7e8a059e29a2b02610aa8aa5c1d65cd45c1a026b
```

Comment:
Maybe include the latest version instead, so we get the recent updates for DDS: `44b6f20dad389471054d8961bd4bf8fab923efdf`

Reply:
I'll make a new PR to include it.